src/lib/isBefore.js

```diff
@@ -1,9 +1,12 @@
-import assertString from './util/assertString';
 import toDate from './toDate';
 
-export default function isBefore(str, date = String(new Date())) {
-  assertString(str);
-  const comparison = toDate(date);
+export default function isBefore(str, options) {
+  // accessing 'arguments' for backwards compatibility: isBefore(str [, comparisonDate])
+  // eslint-disable-next-line prefer-rest-params
+  const comparisonDate = (typeof options === 'object' ? options.comparisonDate : arguments[1]) || Date().toString();
+
+  const comparison = toDate(comparisonDate);
   const original = toDate(str);
```

Comment:
```suggestion
  const original = toDate(date);
```
In line with comment above